Common/Product/SharedProject/ProjectNode.cs

```diff
@@ -2458,6 +2458,8 @@ internal enum EventTriggering {
             } else {
                 ((IDEBuildLogger)this.BuildLogger).OutputWindowPane = output;
```

Comment:
1. cast should be redundant since you've change the type of property to `IDEBuildLogger`
2. (nit) I doubt that we need this protected property since it seems that the only place where `BuildLogger` is access is this class - it can be done as an improvement in a separate PR.